### PR TITLE
feat: Add s3 transport tests for molecule #46

### PR DESCRIPTION
**Pull request by pavelpikta**
* fix fetch from s3 transport
* add s3 transport tests for molecule #46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +130,24 @@ Example Playbook
...
+
+  roles:
+    - role: "lean_delivery.java"
+        java_package: "jre"
+        java_major_version: 8
+        transport: "s3"
+        transport_s3_bucket: "java-molecule-s3-test"
+        transport_s3_path: "/java/jre-8u181-linux-x64.tar.gz"
+        transport_s3_aws_access_key: "{{ lookup('env','AWS_ACCESS_KEY_ID') }}"
+        transport_s3_aws_secret_key: "{{ lookup('env','AWS_SECRET_ACCESS_KEY') }}"
```
Comment (tgadiev) [inline README.md]: Looks like 'overdefining'. Env variables are enough for AWS access.

File: molecule/cloud-aws-direct-s3/molecule.yml [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  name: ansible
+  inventory:
+    group_vars:
+      transport_s3:
+        java_package: "jre"
+        java_major_version: 8
+        transport: "s3"
+        transport_s3_bucket: "java-molecule-s3-test"
+        transport_s3_path: "/java/jre-8u181-linux-x64.tar.gz"
+        transport_s3_aws_access_key: "{{ lookup('env','AWS_ACCESS_KEY_ID') }}"
```
Comment (tgadiev) [inline molecule/cloud-aws-direct-s3/molecule.yml]: I'd remove these lines and set corresponding env parameters below instead.

File: molecule/cloud-aws-direct-s3/molecule.yml [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        transport_s3_aws_secret_key: "{{ lookup('env','AWS_SECRET_ACCESS_KEY') }}"
+  log: False
+  playbooks:
+    create: ../resources/provisioning/AWS/create.yml
+    prepare: ../resources/provisioning/AWS/prepare.yml
```
Comment (tgadiev) [inline molecule/cloud-aws-direct-s3/molecule.yml]: May be change prepare playbook only for this scenario?

Comment (pavelpikta) [reply molecule/cloud-aws-direct-s3/molecule.yml]: resolve in eda0ef1

**Pull request merged**